lock-api/src/main/java/com/palantir/lock/watch/LockWatchEventCache.java

```diff
@@ -42,14 +42,26 @@ void processGetCommitTimestampsUpdate(
             Collection<TransactionUpdate> transactionUpdates, LockWatchStateUpdate update);
 
     /**
-     * Updates the cache with the update, and calculates the {@link CommitUpdate} taking into account all changes to
-     * lock watch state since the start of the transaction, excluding the transaction's own commit locks.
+     * Calculates the {@link CommitUpdate} taking into account all changes to lock watch state since the start of the
+     * transaction, excluding the transaction's own commit locks.
      *
      * @param startTs start timestamp of the transaction
      * @return the commit update for this transaction's precommit condition
      */
     CommitUpdate getCommitUpdate(long startTs);
```

Comment:
When are we actually going to call this and not the other one? Is it just for atlas-proxy?

Reply:
Yes!